### PR TITLE
Fix card border

### PR DESCRIPTION
**Pull request by Nauna**
goes with https://github.com/nanoPayinc/NANOPAY/pull/11490

### CODE REVIEW

File: src/foam/u2/view/CardSelectView.js [hunk 1]
```diff
@@ -43,6 +43,15 @@ foam.CLASS({
     ^innerFlexer {
       min-width: -webkit-fill-available;
     }
+    .foam-u2-borders-CardBorder {
```
Comment (kgrgreer) [inline src/foam/u2/view/CardSelectView.js]: Should be scoped, otherwise it will still ge global.

Comment (Nauna) [reply src/foam/u2/view/CardSelectView.js]: :(

**Comment by kgrgreer** [comment]
Did you test this? Should there be a space after the ^?

File: src/foam/nanos/crunch/ui/MinMaxCapabilityWizardlet.js [hunk 2]
```diff
@@ -121,15 +121,6 @@ foam.CLASS({
   ],
 
   methods: [
-    function createView(data) {
```
Comment (Nauna) [inline src/foam/nanos/crunch/ui/MinMaxCapabilityWizardlet.js:124]: @KernelDeimos @tharmaman 
Just an fyi with this. Tested

Comment (KernelDeimos) [reply src/foam/nanos/crunch/ui/MinMaxCapabilityWizardlet.js:124]: Thanks! This was outdated

**Pull request merged**